resources/config/config.yaml

```diff
@@ -95,7 +94,7 @@ mem_mb:
 
 host_filter:
   db_dir: resources/db/bt2
-  accn: GCA_000001405.28
+  accn: GCA_000005845.2
```

Comment:
It seems a bit dangerous to leave an E. coli accession number as a default here. Like, if someone accidentally forgets to change this and they remove human reads from mouse metagenomic samples, it probably isn't a huge deal to just remove them after.  However, I don't imagine anyone would want to remove reads from E. coli in this way. 

I'm just especially wary since this is what caused an issue for the current sample set.